packages/react-components/src/DocsComponents/isHosted.ts

```diff
@@ -0,0 +1,10 @@
+const getURLParameter = (p: string) => {
+  const url = window.location.search.substring(1);
+  const params = url.split('&');
+  const match = params.find(param => {
```

Comment:
since we no longer support IE11... could we use the built in url search param parser ? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams